src/pages/commits/BuildVariantAccordionContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { Accordion } from "components/Accordion";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordionContainer: React.FC<Props> = ({
+  buildVariants,
+}) => (
+  <ColumnContainer>
+    {buildVariants.map(({ displayName }) => (
+      <div>
+        <AccordionContainer key={displayName}>{displayName}</AccordionContainer>
```

Comment:
nit: since this component renders a title, can we call it <BuildVariantName /> instead?